resources/views/sensordata/index.blade.php

```diff
@@ -0,0 +1,55 @@
+@extends('layouts.dashboard')
+
+@section('title', 'Sensor Data')
+
+@section('content')
+<div class="container">
+	<div class="page-header">
+		<h3>Sensor</h3>
```

Comment:
Header should be "Sensor Data" instead of "Sensor".

Reply:
fixed in 86de04d